tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -37,12 +37,12 @@ public function iSaveAnArticle()
 	}
 
 	/**
-	 * @Then I should see the :arg1 message
+	 * @Then I wait for the title :title and  see the message :message
 	 */
-	public function iShouldSeeTheMessage($message)
+	public function iShouldSeeTheMessage($title, $message)
 	{
 		$I = $this;
-		$I->waitForPageTitle('Articles');
+		$I->waitForPageTitle($title);
```

Comment:
Ok. Thnaks Javi. Getting you. I misunderstood it before. I will do it in two steps.